src/main/java/com/redhat/devtools/lsp4ij/settings/ui/LanguageServerPanel.java

```diff
@@ -232,4 +243,26 @@ public ComboBox<ServerTrace> getServerTraceComboBox() {
     public ComboBox<ErrorReportingKind> getErrorReportingKindCombo() {
         return errorReportingKindCombo;
     }
+
+    public @NotNull List<ValidationInfo> doValidateAll() {
+        List<ValidationInfo> validations = new ArrayList<>();
+        var serverNameWidget = getServerName();

```

Comment:
@angelozerr what do you think about adding `@NotNull` to the getServerName and getCommandLine?

Would be cleaner to just call them like this:
```java
getServerName().validate(validations);
getCommandline().validate(validations)
```

Reply:
Server name and command can be null when language server definition comes from an extension point declared in an ij plugin